Orange/widgets/data/owfeaturestatistics.py

```diff
@@ -807,7 +807,7 @@ def __restore_selection(self):
     def __restore_sorting(self):
         """Restore the sort column and order from saved settings."""
         sort_column, sort_order = self.sorting
-        if sort_column < self.model.columnCount():
+        if self.data is not None and sort_column < self.model.columnCount():
```

Comment:
So `__restore_sorting` does nothing if self.data is None. In that case maybe it should be called only when self.data is not None (in the `set_data` method above)? Is the same true for `__restore_selection`?

Reply:
I thought about putting the `None` check in `set_data`, but decided to put it here instead to keep the `__restore_*` methods consistent. The other methods properly handle the case where there is no data, only this one fails. It seemed cleaner to just make them all work.